src/components/Drawer/mdc-temporary-drawer.vue

```diff
@@ -0,0 +1,114 @@
+<!-- thanks to petejohanson for this implementation -->
+
+<template>
+  <aside class="mdc-temporary-drawer mdc-typography" :class="classes">
+    <nav ref="drawer" class="mdc-temporary-drawer__drawer">
+
+      <div class="mdc-temporary-drawer__toolbar-spacer" v-if="$slots['toolbar-spacer'] || toolbarSpacer">
+        <slot name="toolbar-spacer"/>
```

Comment:
This is typically where I'm uncomfortable with the design: should we go for slots or extra markup to encapsulate headers ? slots is a very powerfull feature. but I'm afraid it can be confusing for final user and may limit customization ( had trouble using slots with mdc-card ...) On the other hand, extra markdown can be tedious ... hard to decide if we should go all for slots or all for markup or a mix...

That being said it seems to work for drawer. let's keep it like this for now